syntax = "proto2";
package E2_protobuf;

message SimpleMessage {
    required int32 lucky_number = 1;
}

message E2_request {
    required uint32 req_id = 1;
}

message E2_dummy_response {
    required uint32 req_id = 1;
    required string mess_string = 2;
    required bool result = 3;
}

message gnbIndication{
    required string gnb_id = 1;
    required string general_mess = 2;
}